### DISCUSSION TITLE
pip installs NEMO instead of nemo ?

### DISCUSSION THREAD

**Discussion by dgoryeo** [NVIDIA-NeMo/NeMo#6560] [Q&A] [answered]
Hi,

I keep getting error message: 

```
import nemo
ModuleNotFoundError: No module named 'nemo'
```

Although I have installed nemo through pip. When I look into the packages installed (pip freeze), I see that nemi is listed as: `NEMO==4.5.4.` 

What am I doing wrong? Any workaround?

**Comment by entn-at** [comment, marked as answer]
Did you use `pip install nemo_toolkit`? NEMO appears to be a different package: https://github.com/usnistgov/NEMO

**Reply by dgoryeo** [reply]
That was it, thanks!

**Reply by utkarsh0509** [reply]
still not working 
when I am trying to run python api.py, I am getting this error --
RuntimeWarning: Couldn't find ffmpeg or avconv - defaulting to ffmpeg, but may not work
  warn("Couldn't find ffmpeg or avconv - defaulting to ffmpeg, but may not work", RuntimeWarning)
ModuleNotFoundError: No module named 'nemo'
